### DISCUSSION TITLE
error po.es

### DISCUSSION THREAD

**Discussion by brandall2021** [horilla-opensource/horilla#280] [General]
hola me gustaria informar este error 

Execution of msgfmt failed: /home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:10081:2: syntax error
/home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:10617: end-of-line within string
/home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:11100: end-of-line within string
msgfmt: found 3 fatal errors
CommandError: compilemessages generated one or more errors.
el archivo es.po tiene 3 errores. y si necesita ayuda para traducir en la parte español me avisa

**Comment by horilla-opensource** [comment]
> hola me gustaria informar este error
> 
> Execution of msgfmt failed: /home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:10081:2: syntax error /home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:10617: end-of-line within string /home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:11100: end-of-line within string msgfmt: found 3 fatal errors CommandError: compilemessages generated one or more errors. el archivo es.po tiene 3 errores. y si necesita ayuda para traducir en la parte español me avisa

Hi @brandall2021 ,
Thanks for bringing it out to our notice. We'll update the po files to fix this.
We are open to any kind of contribution to the system. If you find anything more needs to be added to the translations or if anything is wrong, please do feel free to raise it out. 

With Regards,
**Team Horilla**

**Comment by brandall2021** [comment]
hola te  envio la traduccion al español en un 80%
suerte

El lun, 12 ago 2024 a la(s) 5:26 a.m., Horilla ***@***.***)
escribió:

> hola me gustaria informar este error
>
> Execution of msgfmt failed:
> /home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:10081:2:
> syntax error
> /home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:10617:
> end-of-line within string
> /home/brandall/horilla/horilla/locale/es/LC_MESSAGES/django.po:11100:
> end-of-line within string msgfmt: found 3 fatal errors CommandError:
> compilemessages generated one or more errors. el archivo es.po tiene 3
> errores. y si necesita ayuda para traducir en la parte español me avisa
>
> Hi @brandall2021 <https://github.com/brandall2021> ,
> Thanks for bringing it out to our notice. We'll update the po files to fix
> this.
> We are open to any kind of contribution to the system. If you find
> anything more needs to be added to the translations or if anything is
> wrong, please do feel free to raise it out.
>
> With Regards,
> *Team Horilla*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/horilla-opensource/horilla/discussions/280#discussioncomment-10311305>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUW6SLXWNMZD5DXSXLBUK73ZRBWT5AVCNFSM6AAAAABMIPEIPGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMZRGEZTANI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
IT Manager  Linux Admin-Security   PM:56489
IT & Quality Manager  PM:67908
DBA & UNIX System Engineer
http://www.linuxsecurity.com/
